ChatQnA/microservice/xeon/README.md

```diff
@@ -133,9 +202,9 @@ print(embedding)
 Then substitute your mock embedding vector for the `${your_embedding}` in the following cURL command:
 
 ```bash
-curl http://${host_ip}:7000/v1/retrieval\
+curl http://${host_ip}:7000/v1/retrieval \
   -X POST \
-  -d '{"text":"What is the revenue of Nike in 2023?","embedding":${your_embedding}}' \
+  -d '{"text":"What is the revenue of Nike in 2023?","embedding":"'"${your_embedding}"'"}' \
```

Comment:
We just replace the {your_embedding} with the embedding data list, it's good. You did it by export 'your_embedding' environment, right?

Reply:
Yes, that is correct. By doing it with the environment variable, it keeps the command short and readable on the screen, hence did that.

Reply:
@lvliang-intel - any other information or details needed from me? please let me know.